config/prow/plugins.yaml

```diff
@@ -521,6 +521,18 @@ require_matching_label:
     If a SIG or subproject determines this is a relevant issue, they will accept it by applying the `triage/accepted` label and provide further guidance.
 
     The `triage/accepted` label can be added by org members by writing `/triage accepted` in a comment.
+- missing_label: needs-triage
+  org: kubernetes
+  repo: website
+  issues: true
+  regexp: ^triage/accepted$
+  missing_comment: |
+    This issue is currently awaiting triage.
+
+    SIG Docs takes a lead on issue triage for this website, but any Kubernetes member can accept issues by applying the `triage/accepted` label.
+
+    The `triage/accepted` label can be added by org members by writing `/triage accepted` in a comment.
+
```

Comment:
@habibrosyad :wave: Thanks for the PR! Can you remove this newline here? The text after the `missing_comment` line is rendered literally so the newline will also get added to all bot comments in the current form.

Reply:
Sorry, missing that one, thanks @nikhita. Done in https://github.com/kubernetes/test-infra/commit/f3e7e8d79ad9db08cb23afe0b5ae6046ca38ede6.